.github/ISSUE_TEMPLATE/mentee-application.yml

```diff
@@ -0,0 +1,42 @@
+name: Mentee application
+description: Apply to take part in the astropy mentoring scheme as a mentee
+title: "Mentee application - <your name>"
+labels: ["application"]
+assignees:
+  - ''
+body:
+  - type: input
+    id: name
+    attributes:
+      label: Name
+      description: Please give your name
+    validations:
+      required: false
+  - type: input
+    id: usernames
+    attributes:
+      label: GitHub/Slack handles
+      description: Please give the username you use on GitHub and Slack (if they are different) so that we know who to associate the application with.
+    validations:
+      required: true
+  - type: textarea
+    id: role
+    attributes:
+      label: Which role would you like to apply for and why
+      description: The aim of the mentoring program is to eventually enable you to join the astropy team. Let us know which role you are interested in and what interests you about it. This will help us find you a suitable mentor. -->
```

Comment:
```suggestion
      description: The aim of the mentoring program is to eventually enable you to join the astropy team. Let us know which role you are interested in and what interests you about it. This will help us find you a suitable mentor.
```